decidim-core/app/cells/decidim/card_metadata_cell.rb

```diff
@@ -52,5 +53,55 @@ def space_item
         url: Decidim::ResourceLocatorPresenter.new(participatory_space).path
       }
     end
+
+    def start_date
+      nil
+    end
+
+    def end_date
+      nil
+    end
+
+    def dates_blank?
+      [start_date, end_date].any?(&:blank?)
+    end
+
+    def current_date
```

Comment:
Should return a date and not a time?

Reply:
I've used Time for a more accurate comparison if the start and end dates include time, but the processes use date only. I think I got confused with the phases, which do include the time.